Update hardware.md (#438)
BitcoinDesign/Guide: guide/getting-started/hardware.md

# Hardware overview

> Generally, any hardware that is connected to the Internet can interact with the Bitcoin network. While personal computers and smartphones work great for general wallet management and transactions, some dedicated hardware products are optimized for the nuances of Bitcoin. We’ll explore them and how they work here.

Reviewer: ```suggestion
Generally, any hardware device that is connected to the Internet can interact with the Bitcoin network. While personal computers and smartphones work great for general wallet management and transactions, some dedicated hardware products are optimized for the nuances of Bitcoin. We’ll explore them and how they work here.
```

## Recovery phrase storage

> Simple, cheap, and robust storage solutions for recovery phrases come in many forms. [Seedplate](https://bitcoinseedbackup.com/){:target="_blank"}, is a metal plate that users can etch their recovery phrase into for long-term offline storage. [CardWallet](https://www.cardwallet.com){:target="_blank"}, by comparison, is the size of a credit card and pre-loaded with a recovery phrase (hidden behind a security seal) and a matching Bitcoin address. For an even simpler solution, a piece of paper will do. Although paper is less durable and easy to lose, it is a strength of Bitcoin that such simple solutions are possible.

Author: Adding comment by @sqcken for future reference or to discuss and improve right away in a review process.

> Feels like we should be saying more and covering more recovery storage solutions? No mention of Cobo type steel wallets?

Reviewer: ```suggestion
Simple, cheap, and robust storage solutions for recovery phrases come in many forms. [Seedplate](https://bitcoinseedbackup.com/){:target="_blank"}, is a metal plate that users can etch their recovery phrase into for long-term offline storage. For an even simpler solution, a piece of paper will do. Although paper is less durable and easy to lose, it is a strength of Bitcoin that such simple solutions are possible.
```
I would remove the point on CardWallet, just read their website and the private keys are generated by the facility, not by the user, major security vulnerability. https://www.cardwallet.com/en/faq/ 

If you wanted to add another point, just for fun and not recommended of course, you could say the human brain is also a hardware recovery phrase storage option 😆

Author: cc @GBKS

Reviewer 2: For Ken's comment, I don't see a huge difference between Seedplate and the Cobo steel wallet. In one, you etch the words, on the other you put together small metal pieces with characters inscribed, but either way it's a steel backup.

Good point about removing the Cardwallet.

How about we link this to the ["Single metal backup" section of the "Bitcoin backups" page](https://bitcoin.design/guide/private-key-management/backups/#single-metal-backup), which is a hands-on guide for specifically these products. We could then add more links to reference products on that page, which would be appropriate for a hands-on guide, IMHO.

Author: Good catch.

Do you have a concrete suggestion here? Do we just re-write this section to mention paper and metal that link back to the bitcoin backups page which indeed provides good info

Reviewer: I reckon we could just say something a long the lines of "various metal plate seed storage products exists and they are good for xyz reason." No need to mention seedplate / cobo plate - they are all essentially the same.

Reviewer 2: As this change/conversation was outdated, I created a new change suggestion with the link to the backups page.

# Hardware overview

Generally, any hardware device that is connected to the Internet can interact with the Bitcoin network. While personal computers and smartphones work great for general wallet management and transactions, some dedicated hardware products are optimized for the nuances of Bitcoin. We’ll explore them and how they work here.

## Recovery phrase storage

Simple, cheap, and robust storage solutions for recovery phrases come in many forms.

> [Seedplate](https://bitcoinseedbackup.com/){:target="_blank"}, is a metal plate that users can etch their recovery phrase into for long-term offline storage. For an even simpler solution, a piece of paper will do. Although paper is less durable and easy to lose, it is a strength of Bitcoin that such simple solutions are possible.

Reviewer 2: ```suggestion
[Seedplate](https://bitcoinseedbackup.com/){:target="_blank"}, is a metal plate that users can etch their recovery phrase into for long-term offline storage. For an even simpler solution, a piece of paper will do. Although paper is less durable and easy to lose, it is a strength of Bitcoin that such simple solutions are possible. See [Bitcoin backups]({{ '/guide/private-key-management/backups/' | relative_url }}) for more details.
```

   height = 400
%}

> Bitcoin hardware wallets play a similar role but with several noteworthy differences. Like key fobs, they securely store an authentication method and isolate it from the Internet, networks, and other devices. In bitcoin, these are the seed, associated keys, and functions that require interaction with them.

Reviewer: ```suggestion
Bitcoin hardware wallets play a similar role but with several noteworthy differences. Like key fobs, they securely store an authentication method and isolate it from the Internet, networks, and other devices. Bitcoin hardware wallets store private keys, seeds, wallet descriptors and other data that allows you to interact with a Bitcoin wallet application externally and construct Bitcoin transactions.
```

Felt like this was a little ambiguous, added some suggestions.

Author: Too much complexity, user doesn't know what descriptors are, we barely introed them to BTC, this is getting started page. I mean "wallet descriptors" is way too complicated for this section.

Reviewer 2: We really need to get #59 going and create that page about hardware wallets where we can add these details, and then link to them.

> Because hardware wallets are so optimized for this specific use case, they have tiny screens and limited input options. Most interactions with them happen via desktop software that the manufacturer provides. The application sends only specific requests to the hardware wallet to verify or sign data. This provides a strong balance of security and convenience and keeps hardware wallets generally affordable.

Reviewer: ```suggestion
Because hardware wallets are so optimized for this specific use case, they have tiny screens and limited input options. Most interactions with them happen via desktop software that the manufacturer provides. The application sends only specific requests to the hardware wallet to verify or sign data. This provides a strong balance of security and convenience.
```

Not sure why this makes them more affordable?

Author: Agreed "affordable" also depends on who's looking at it. Removed.

## Nodes

> Thousands of computers participate in the Bitcoin network. They are called nodes. Nodes exchange, verify, and compare block and transaction data.

Reviewer: ```suggestion
Tens of thousands of computers participate in the Bitcoin network. They are called nodes. Nodes exchange, verify, and compare block and transaction data.
```

Could add a reference here to this: https://luke.dashjr.org/programs/bitcoin/files/charts/software.html

> Running your own node, instead of relying on a third-party node, lets you validate transactions on your own device. Nodes also improve privacy and help secure the network. Several DIY and consumer hardware products make it easy to run your own node.

Reviewer: ```suggestion
Running your own node, instead of relying on a third-party node, lets you validate transactions on your own device. It also improves the privacy of your transactions and help secure the network. Several DIY and consumer hardware products make it easy to run your own node.
```

Reviewer 2: We should add a link to the [Glossary node page](https://bitcoin.design/guide/glossary/node/) in a separate PR.

Author: Added https://github.com/BitcoinDesign/Guide/pull/438/commits/6a36ce616e679c1b4925d434411b8d3d550b29fd

   height = 400
%}

> Dedicated node hardware is particularly relevant for heavy users of the Lightning network. While a bitcoin node can be turned off and on, Lightning nodes need to be online at all times to maintain payment channels with other nodes. Smartphones are typically online 24/7 but are not suited for this task due to large amounts of data transfer and storage capacity requirements.

Reviewer: ```suggestion
Dedicated node hardware is particularly relevant for users of the Lightning network. While a bitcoin node can be turned off and on, Lightning nodes need to be online at all times to maintain payment channels with other nodes. Smartphones are typically online 24/7 but are not suited for this task due to large amounts of data transfer and storage capacity requirements.
```

We should add an entry for Lightning nodes to this page in a separate PR and call this section a Bitcoin Node. There is a lot of nuances here to consider, for example you can run a Lightning node and use something like Neutrino to retrieve blockchain data.

Author: LN is out of scope of this PR. Just doing minor changes here.

Reviewer 2: I'd just keep it simple and link to the [Glossary node page](https://bitcoin.design/guide/glossary/node/), and cover all the Bitcoin/Lightning node details there.

Author: Done.
Added a bunch of links to that awesome page https://github.com/BitcoinDesign/Guide/pull/438/commits/6a36ce616e679c1b4925d434411b8d3d550b29fd

While plug-and-play hardware nodes can solve various problems for bitcoin users, they have not seen widespread adoption as commercial products.

## Miners

> Bitcoin has become a professional undertaking. Today, dedicated hardware (so-called ASICS, or application-specific integrated circuits) are optimized for continuously executing the Bitcoin hashing algorithm (called [SHA-256](https://en.wikipedia.org/wiki/SHA-2){:target="_blank"}) over and over until an acceptable hash is found. A modern miner (in 2020) can generate around 14,000,000,000,000 hashes per second. The more hashes generated, the higher that miner’s chance of “finding” a new block and receiving newly minted bitcoin as a reward.

Reviewer: ```suggestion
Since Bitcoins inception a multi-billion dollar industry has emerged around the mining of new bitcoins. Today, dedicated hardware (so-called ASICS, or application-specific integrated circuits) are optimized for continuously executing the Bitcoin hashing algorithm (called [SHA-256](https://en.wikipedia.org/wiki/SHA-2){:target="_blank"}) over and over until an acceptable hash is found. A modern miner (in 2020) can generate around 14,000,000,000,000 hashes per second. The more hashes generated, the higher that miner’s chance of “finding” a new block and receiving newly minted bitcoin as a reward.
```

Reviewer 2: I think it's important to keep the word "mining" in the first sentence. Bitcoin itself is not a professional undertaking, but mining bitcoin is. So I agree with this latest tweak by @Bosch-0 .

   height = 400
%}

> Outside of purchasing hardware, the main cost of mining is electricity. That’s why mining facilities are found wherever electricity is abundant and cheap, such as in Iceland (geothermal) or in China (hydroelectricity).

Reviewer: There is also a big benefit in that Bitcoin mining can turn localized / waste electricity into bitcoin (value).

Reviewer 2: Linking to some additional material could be helpful here. I liked Nic Carters presentation at the B word conference, he did a very thorough analysis there. Unfortunately, the conference videos are not publicly available. Maybe we can link to the [Square/Ark paper](https://assets.ctfassets.net/2d5q1td6cyxq/2D2BnksJjavw4a6SUvAPwZ/c42a9e3a520b0cc3b230cda3b43eead5/BCEI_White_Paper_.pdf)?

Author: Need a concrete suggestion here where to put that link :)

## ATMs

> A Bitcoin ATM (Automated Teller Machine) allows users to purchase or sell bitcoin for cash or with debit cards. These machines are usually bound to local money transmitting laws and regulations, such as KYC (Know Your Customer) and anti-money laundering regulations. More on [Wikipedia](https://en.wikipedia.org/wiki/Bitcoin_ATM).

Reviewer: ```suggestion
A Bitcoin ATM (Automated Teller Machine) allows users to purchase or sell bitcoin for cash or with debit cards. These machines are usually bound to local money transmitting laws and regulations, such as KYC (Know Your Customer) and AML (anti-money laundering) regulations. More on [Wikipedia](https://en.wikipedia.org/wiki/Bitcoin_ATM).
```
